### DISCUSSION TITLE
How to hide the "make-public“ tag from my public garden page?

### DISCUSSION THREAD

**Discussion by tombarys** [roam-garden/i#8] [Q&A] [answered]
I want to hide the tag, which makes the page publicly visible. Is there an easy way to do it? 

I tried to hide the block with the tag using the "hide-the-block-tag" set up via settings but it hides the whole page as the make-public tag is not only visible but "missing internally".

**Comment by Stvad** [comment, marked as answer]
Copying previous answers to this from Mike Lesiuk:

You can use the following CSS to hide the tag  (changing it to match the tag you use)
```
a.garden-hashtag-link[href^="/make-public"] {
  display:none;
}
```

**Comment by Stvad** [comment]
Another approach by @kalmir (allows hiding whole block):

>It hides the last item of the last list. If thats the bullet where you put your make_public tag, it is gonna be hidden.

```
#gatsby-focus-wrapper > div > div > div > div > div > ul:last-of-type > li:last-of-type {
    display: none;
}
```

I imagine this can be adopted to hide first item in the list too

**Reply by eurunuela** [reply]
What if it's the first item in the page?

**Reply by kalmir** [reply]
Try `first-of-type` like this:

`#gatsby-focus-wrapper > div > div > div > div > div > ul:first-of-type > li:first-of-type {
    display: none;
}`

**Reply by eurunuela** [reply]
That worked perfectly. Thanks!

**Comment by tombarys** [comment]
Thanks!
Tomáš

**Comment by ghansali** [comment]
That's how I hide it with my CSS:

a[title="hashtag_make-public"] {
    display: none;
}

**Comment by Stvad** [comment]
With the new structure (after #23 ) the following should allow you to hide the first block on the page:
```css
.note-content > .block:first-of-type {
    display: none;
}
```

or similarly for the last block:

```css
.note-content > .block:last-of-type {
    display: none;
}
```
